validator/db/filesystem/import_test.go

```diff
@@ -91,6 +91,8 @@ func TestStore_ImportInterchangeData_BadFormat_PreventsDBWrites(t *testing.T) {
 			"Imported proposal signing root is different than the empty default",
 		)
 	}
+
+	t.Log("TestStore_ImportInterchangeData_BadFormat_PreventsDBWrites (filesystem/import_test.go) passed")
 }
```

Comment:
Is this log for test purpose? or intended to leave?

Reply:
It seems some tests are failed, but in fact they have such log:
  - No pass/skip/fail event found for test

These tests are solved by adding a `t.Log` statement manually.